### PR TITLE
Revert to 4.8.3 no refunds rc.1

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5149

### Change description ###
Revert commits from toolkit master for refund release failure.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/shared/components/workbasket-filters/workbasket-filters.component.ts [hunk 1]
```diff
@@ -320,18 +320,16 @@ export class WorkbasketFiltersComponent implements OnInit {
...
-        if (formGroupLS[REGION_LIST_AND_FRC_FILTER] !== this.formGroup.get(REGION_LIST_AND_FRC_FILTER).value) {
-          for (const key in formGroupLS) {
-            if (formGroupLS.hasOwnProperty(key)) {
-              const value = formGroupLS[key];
-              // Clear the filter form group control values if it has a value in local storage
-              // The regionList form group control value should be ignored as it always contain the latest value
-              if (key !== REGION_LIST_AND_FRC_FILTER && value != null) {
-                this.formGroup.get(key).setValue(null);
-              }
+      // If regionList value does not match between local storage and form group
```
Comment (johnbenjamin-hmcts) [inline src/shared/components/workbasket-filters/workbasket-filters.component.ts:323]: Do we need to revert this? I remember Lee did this fix when working on a bug.

**Review by johnbenjamin-hmcts — CHANGES_REQUESTED** [review]
@RiteshHMCTS I added one comment. Could you please check with Lee?

File: src/shared/components/workbasket-filters/workbasket-filters.component.ts (hunk 1 above)
Comment (RiteshHMCTS) [reply src/shared/components/workbasket-filters/workbasket-filters.component.ts:323]: @johnbenjamin-hmcts I have reverted back to the version which is as per revert-to-4.8.3-no-refunds-rc.1

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]
@RiteshHMCTS thanks

**Pull request merged**